src/app/components/Login.js

```diff
@@ -1,48 +1,41 @@
 import React from 'react';
-import firebase from "../firebaseConfig";
+import firebase from '../firebaseConfig';
 import * as NewsActions from '../actions/NewsActions';
 
 export default class Login extends React.Component {
 
-  // storeUserProfile(user) {
-  //   window.localStorage.setItem('uid', user.uid);
-  //   window.localStorage.setItem('email', user.email);
-  //   window.localStorage.setItem('name', user.displayName);
-  // }
-
   componentDidMount() {
      // FirebaseUI config.
-     const uiConfig = {
-		'signInFlow': 'popup',
-		'signInSuccessUrl': '/',
-		'signInOptions': [
-			firebase.auth.GoogleAuthProvider.PROVIDER_ID,
-		],
-		'tosUrl': 'https://www.google.com',
-		'callbacks': {
-			'signInSuccess': (user, credential, redirectUrl) => {
-        console.log('User just signed in');
-        window.localStorage.setItem('uid', user.uid);
-        window.localStorage.setItem('email', user.email);
-        window.localStorage.setItem('name', user.displayName);
-        window.localStorage.setItem('photo', user.photoURL);
-        return true;
-			}
-		}
-	};
+    const uiConfig = {
+      signInFlow: 'popup',
+      signInSuccessUrl: '/',
+      signInOptions: [
+        firebase.auth.GoogleAuthProvider.PROVIDER_ID,
+      ],
+      tosUrl: 'https://www.google.com',
+      callbacks: {
+        signInSuccess: (user, credential, redirectUrl) => {
+          console.log('User just signed in');
+          window.localStorage.setItem('uid', user.uid);
+          window.localStorage.setItem('email', user.email);
+          window.localStorage.setItem('name', user.displayName);
+          window.localStorage.setItem('photo', user.photoURL);
```

Comment:
Cool, but could you explore setting this items in localstorage  as a single Object/item and use some sort of name unique to your app ?